lib/utils.pm

```diff
@@ -420,9 +420,9 @@ sub sle_version_at_least {
           && !check_var($version_variable, '12-SP3');
     }
 
-    if ($version eq '13') {
-        return sle_version_at_least('12-SP4', version_variable => $version_variable)
-          && !check_var($version_variable, '13');
+    if ($version eq '15') {
+        return sle_version_at_least('12-SP3', version_variable => $version_variable)
+          && !check_var($version_variable, '15');
```

Comment:
> From the customers point of view and our point of view it should be "higher than any 12-something version"

Does the above statement still hold true when both SLE 12 SP3 and SLE 12 SP4 will be mandatory base_systems for SLE 15 migration testing?